### DISCUSSION TITLE
Settings / player list presentation improvements

### DISCUSSION THREAD

**Discussion by madbrain76** [music-assistant/support#2511] [Feature requests and ideas]
Currently, the players in the settings / player list are sorted by alphabetical order.
This is not ideal when there are numerous disabled players,that still show up in the middle of the list. The only way to hide them is to delete the corresponding provider, but that's often not suitable - thus the existence of the "disable" feature on individual players.

I propose a simple change to show two lists, first the enabled players, followed by some separator line, followed by the list of disabled players.
A possible alternative would be some sort of toggle button to switch between the lists of enabled and disabled players.

The next proposal concerns groups. Currently, there is no visual distinction between Airplay groups and other players, for example. This makes them hard to locate in a long player list. The same issue exists for Chromecast groups - one can't tell them apart from individual players.
Universal groups already have their own icon. I'm not sure what the difference should look like. It could be a different background color for the icon. Or there could be a separate column in the view indicating that the device is a group. Or the player type string should include the information. For example, it could simply say "Airplay group" , or "Chromecast group",

**Comment by OzGav** [comment]
Not discounting your suggestion generally but you can set your own icon for players in the settings such as
<img width="285" alt="image" src="https://github.com/music-assistant/support/assets/19848947/bf371ba0-166d-44c6-8fef-97a583a15090">

**Comment by madbrain76** [comment]
Thanks ! That is helpful for the second part of the request.

**Reply by OzGav** [reply]
I am thinking the UI changes since this request may make it redundant?

**Reply by OzGav** [reply]
Indeed this request has been implemented in the latest versions of MA
